packages/framer-motion/src/value/index.ts

```diff
@@ -317,6 +317,11 @@ export class MotionValue<V = any> {
         return this.current
     }
 
+
```

Comment:
Same as `play`/`pause` - preferable just to interface with the animation directly.

Reply:
I've updated the code to access the animation directly, if you had another way in mind to accomplish this please let me know!